collective/trustedimports/suds.py

```diff
@@ -0,0 +1,3 @@
+from collective.trustedimports.util import whitelist_module
+
+whitelist_module('suds')
```

Comment:
Looking at this documentation this accepts urls which makes it a security risk.
I think what we need to do is have a env variable called SAFEIMPORTS_URL_BLACKLIST with a list of urls that should not be allowed. And we patch any function that accesses the internet to check this list first and throw an exception.

 https://bitbucket.org/jurko/suds/wiki/Original%20Documentation. Also you will likely need to whitelist more things that this to make it work.

Reply:
@djay , I implement wildcard in blacklist, using fmatch, the blacklist use semicolon as delimiter